frontend/app/views/support/DisplayText.scala

```diff
@@ -34,23 +34,8 @@ object DisplayText {
       case _ => "Benefits"
     }
 
-    def highlights = tier match {
-      case Friend => List(Highlight(
-        """Receive regular updates from the membership community,
-          | book tickets to Guardian Live and Local events and access the Guardian members area""".stripMargin))
-
-      case Supporter => List(Highlight("Support our journalism and keep theguardian.com free of charge"),
-        Highlight("Get access to tickets and to the live broadcast of events"))
-
-      case Partner => List(
-        Highlight(s"Includes $zuoraFreeEventTicketsAllowance tickets to Guardian Live events (or 4 Guardian-published books) per year", isNew = true),
-        Highlight("Get priority booking and 20% discount on Guardian Live, Guardian Local and most Guardian Masterclasses")
-      )
-
-      case Patron => List(Highlight("Show deep support for keeping the Guardian free, open and independent."),
-        Highlight("Get invited to a small number of exclusive, behind-the-scenes functions")
-      )
-      case _ => Nil
+    def highlights(countryGroup: CountryGroup = UK) = Highlights.forTier(tier).filterNot {
+      highlight => Highlights.marketedOnlyToUK(highlight) && countryGroup != UK
```

Comment:
You mean pass the `countryGroup` to `Benefits.forTier` and `Highlights.forTier`?
and do the `.filterNot()` inside that function rather than in `DisplayText`?